openwisp_monitoring/device/utils.py

```diff
@@ -1,9 +1,15 @@
+from swapper import load_model
+
 from ..db import timeseries_db
 from . import settings as app_settings
 
 SHORT_RP = 'short'
 DEFAULT_RP = 'autogen'
 
+Device = load_model('config', 'Device')
+DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
+DeviceData = load_model('device_monitoring', 'DeviceData')
```

Comment:
@praptisharma28 you are getting the `django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.` error because of these assignments here. 

This file is loaded in the global namespace in openwisp_monitoring/device/apps.py. Thus, the code is trying to load the model before Django app has been loaded. 

I don't see these models being used in this file. Moreover, this file does not look right to store the signal receiver. I would add the logic to a classmethod in AbstractDeviceMonitoring, similar to `handle_disabled_organization` https://github.com/openwisp/openwisp-monitoring/blob/998c81b4a6e7600d418db2fb96e6a975f72b45cc/openwisp_monitoring/device/base/models.py#L438-L455

Reply:
OKay, got the idea. Thanks Gagan.